src/main/com/smithsmodding/armory/compatibility/recipes/anvil/BlacksmithsAnvilRecipeCategory.java

```diff
@@ -46,6 +46,12 @@ public String getTitle() {
         return LOCALENAME;
     }
 
+    @Override
+    public String getModName()
+    {
+        return References.General.MOD_ID;
+    }
+
     @Nonnull
     @Override
     public IDrawable getBackground() {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this left curly brace to the beginning of next line of code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)